themes/godotengine/layouts/home.htm

```diff
@@ -306,10 +306,10 @@ <h2 class="text-center">Sponsored by</h2>
       <a class="sponsor card base-padding" href="https://www.gamblify.com/" target="_blank">
         <img src="{{ 'assets/home/sponsors/gamblify.png' | theme }}" alt="Gamblify">
       </a>
-      <a class="sponsor card" href="https://www.moonwards.com/" target="_blank">
+      <a class="sponsor card  base-padding" href="https://www.moonwards.com/" target="_blank">
```

Comment:
Nitpick, but there should only be one space to separate each CSS class :slightly_smiling_face: